src/routes/api/games.js

```diff
@@ -34,6 +34,24 @@ games.post('/', validator.body(postGameBody), async (req, res) => {
   return status.created(res, { ...newGame });
 });
 
+games.get('/', async (req, res) => {
```

Comment:
This should probably be using a query validator middleware
```suggestion
const getGameQuery = Joi.object({
  pageNum: Joi.number().required(),
  pageSize: Joi.number().required(),
  userEmail: Joi.string().required()
});

games.get('/', validator.query(getGameQuery), async (req, res) => {
```

Not sure off the top of my head if `number()` will work in the query, though, so you may have to call it a string and add a regex

Reply:
You're right. I added though params after I wrote the initial API call and didn't remember to add validation. Apparently GitHub can just commit your suggestion, which is dope, so I'm going to do that and test it out locally. Will merge when it works as expected 👍

Reply:
Looks like `joi` handles string as numbers

![image](https://user-images.githubusercontent.com/11067969/76767640-50ee1600-6770-11ea-9c5d-c437877475af.png)